### PR TITLE
feat(useClipboard): added a new composable

### PR DESCRIPTION
**Pull request by Eyoatam**

### CODE REVIEW

File: docs/composable/web/clipboard.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+```
+
+| State         | Type       | Description                                                                                                                 |
+| ------------- | ---------- | --------------------------------------------------------------------------------------------------------------------------- |
+| supported     | `Boolean`  | Returns true if the browser has [navigator.clipboard](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/clipboard) |
+| clipboardData | `Ref(any)` | data in clipboard                                                                                                           |
+
+## Example
+
+<clipboard-example/>
```
Comment (pikax) [inline docs/composable/web/clipboard.md:22]:
```suggestion
<ClientOnly>
  <clipboard-example/>
</ClientOnly>
```

This will make the PR to pass

**Review by pikax — CHANGES_REQUESTED** [review]
Just that change, thank you 👍

**Comment by pikax** [comment]
I'm failing to get the docs example working on my local environment, I will get in it working in the mean time.

Thank you

**Comment by Eyoatam** [comment]
> I'm failing to get the docs example working on my local environment, I will get in it working in the mean time.
> 
> Thank you

cool 👍🏼

**Comment by pikax** [comment]
> It wasn't working on my local environment either, I Think the problem is the package hasn't been updated 🤔

The reason was you were missing the export on the `index.ts`  


Made a few changes to the API to be a bit more understandable and also added the automatic get of the the clipboard on `focus`

**Comment by pikax** [comment]
Thank you for the PR 👍

**Pull request merged**